docs/core/development/policy/release-process.rst

```diff
@@ -169,111 +151,27 @@ A week is a generally good length of time to wait before doing the final release
 How to do a final release
 -------------------------
 
+
 Prepare the branch
 ~~~~~~~~~~~~~~~~~~
 
 #. Have the release branch, previously used to generate a release candidate, checked out
-#. Run ``python -m incremental.update Twisted``.
-#. Revert the release candidate newsfile changes, in order.
-#. Run ``towncrier`` to make the final newsfile.
-#. Add the quote of the release to the ``README.rst``
-#. Make a new quote file for the next version
-
-   - ``git mv docs/fun/Twisted.Quotes docs/historic/Quotes/Twisted-$API; echo '' > docs/fun/Twisted.Quotes; git add docs/fun/Twisted.Quotes``
-
-#. Commit the version and ``README.rst`` changes.
+#. Manually update the version and realease date. Commit and push.
 #. Submit the ticket for review
 #. Pause until the ticket is reviewed and accepted.
-#. Tag the release.
-
-   - ``git tag -s twisted-$RELEASE -m "Tag $RELEASE release"``
-   - ``git push --tags``
-
-
-Cut the tarballs & installers
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-#. Using a checkout of the release branch or the release tag (with no local changes!), build the tarballs:
-
-   - ``python setup.py sdist --formats=bztar -d /tmp/twisted-release``
-
-#. Build Windows wheel
-
-   - Download the latest ``.whl`` files from `Buildbot <https://buildbot.twistedmatrix.com/builds/twisted-packages/>`_ and save them in the staging directory
-
-#. Sign the tarballs and Windows installers.
```

Comment:
I think it's worth publishing that the releases are no longer signed

Reply:
Yes. I think that this should be top level info for the next release together with the fact that 2.7 is no longer supported.